docs/wiki/Localization-and-Translation.md

```diff
@@ -31,4 +33,4 @@ Once the translations are added to **client/core/locale_ntfn.go**, the new map i
 
 The JavaScript strings involved editing [client/webserver/site/src/js/locales.js](https://github.com/decred/dcrdex/blob/master/client/webserver/site/src/js/locales.js) with a new `export const zhCN` object, with strings corresponding to the English text in the `enUS` object at the top of the same file.  Finally, the new object is listed in the `const localesMap` at the end of the file.
 
-When testing, remember to rebuild the site assets bundle with `npm ci && npm run build` in the **client/webserver/site** folder.
\ No newline at end of file
+When testing, remember to rebuild the site assets bundle with `npm ci && npm run build` in the **client/webserver/site** folder.
```

Comment:
Thank you for re-adding newlines at the end of the files.  This is important for all text files, not just markdown.  https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
https://github.com/decred/dcrd/pull/2941